src/components/icons/Icon.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { BiPlusCircle } from 'react-icons/bi';
+import {
+  MdPictureAsPdf,
+  MdArchive,
+  MdDelete,
+  MdDeleteForever,
+} from 'react-icons/md';
+
+interface Props {
+  type: 'view' | 'new' | 'archive' | 'delete' | 'purge';
+  fontSize?: number;
+  color?: string;
+}
+
+export function Icon(props: Props) {
+  const fontSize = props.fontSize ? props.fontSize : 18;
```

Comment:
This is not the correct naming scheme. The correct prop should be `size` since you are in the context of icons.

Reply:
Changed